.github/workflows/ci.yml

```diff
@@ -51,13 +50,17 @@ jobs:
     # ============
     - name: Dependencies [Windows]
       if: matrix.os == 'windows-latest'
-      shell: bash
       run: |
-        git clone https://github.com/microsoft/vcpkg.git --depth 1 --branch 2019.10 ${VCPKG_ROBOTOLOGY_ROOT}
-        git clone https://github.com/robotology/robotology-vcpkg-binary-ports.git ${VCPKG_ROBOTOLOGY_BIN_PORTS_ROOT}
-        ${VCPKG_ROBOTOLOGY_ROOT}/bootstrap-vcpkg.sh
-        ${VCPKG_ROBOTOLOGY_ROOT}/vcpkg.exe --overlay-ports=${VCPKG_ROBOTOLOGY_BIN_PORTS_ROOT} install --triplet x64-windows ace freeglut gsl eigen3 opencv[core,contrib] \
-                                                                                                                            vtk ipopt-binary matio hdf5 fftw3
+        # To avoid spending a huge time compiling vcpkg dependencies, we download a root that comes precompiled with all the ports that we need 
+        choco install -y wget unzip
+        # To avoid problems with non-relocatable packages, we unzip the archive exactly in the same C:/robotology/vcpkg 
+        # that has been used to create the pre-compiled archive
+        cd C:/
+        md C:/robotology
+        md C:/robotology/vcpkg
+        wget https://github.com/robotology/robotology-superbuild-dependencies-vcpkg/releases/download/${env:vcpkg_robotology_TAG}/vcpkg-robotology.zip
+        unzip vcpkg-robotology.zip -d C:/robotology/vcpkg
+        # ${VCPKG_INSTALLATION_ROOT}/vcpkg.exe install --triplet x64-windows opencv3[contrib] vtk hdf5 fftw3
```

Comment:
@traversaro if I enable this line, I'll get a CI error.
See https://github.com/robotology/assistive-rehab/runs/531463623?check_suite_focus=true

Reply:
By default on Windows the shell is powershell, so you need to use env variables as `${env:VCPKG_INSTALLATION_ROOT}`

Reply:
Thanks!